[2024-11-02T21:30:19.353+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: table_dag.submit_tables scheduled__2024-10-23T00:00:00+00:00 [queued]>
[2024-11-02T21:30:19.426+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: table_dag.submit_tables scheduled__2024-10-23T00:00:00+00:00 [queued]>
[2024-11-02T21:30:19.434+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-11-02T21:30:19.626+0000] {taskinstance.py:1380} INFO - Executing <Task(PinotTableSubmitOperator): submit_tables> on 2024-10-23 00:00:00+00:00
[2024-11-02T21:30:19.662+0000] {standard_task_runner.py:57} INFO - Started process 12862 to run task
[2024-11-02T21:30:19.949+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'table_dag', 'submit_tables', 'scheduled__2024-10-23T00:00:00+00:00', '--job-id', '267', '--raw', '--subdir', 'DAGS_FOLDER/table_dag.py', '--cfg-path', '/tmp/tmpctns4aej']
[2024-11-02T21:30:20.005+0000] {standard_task_runner.py:85} INFO - Job 267: Subtask submit_tables
[2024-11-02T21:30:20.336+0000] {task_command.py:415} INFO - Running <TaskInstance: table_dag.submit_tables scheduled__2024-10-23T00:00:00+00:00 [running]> on host f79a3875bd34
[2024-11-02T21:30:20.701+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ctwang' AIRFLOW_CTX_DAG_ID='table_dag' AIRFLOW_CTX_TASK_ID='submit_tables' AIRFLOW_CTX_EXECUTION_DATE='2024-10-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-23T00:00:00+00:00'
[2024-11-02T21:30:20.782+0000] {pinot_table_operator.py:31} ERROR - Failed to submit table: 409 - {"code":409,"error":"Table config for customer_dim_OFFLINE already exists. If this is unexpected, try deleting the table to remove all metadata associated with it."}
[2024-11-02T21:30:20.804+0000] {pinot_table_operator.py:35} ERROR - An error occurred: Table submission failed with status code 409
[2024-11-02T21:30:20.930+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=table_dag, task_id=submit_tables, execution_date=20241023T000000, start_date=20241102T213019, end_date=20241102T213020
[2024-11-02T21:30:21.109+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-11-02T21:30:21.268+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
